avocado/plugins/remote.py

```diff
@@ -68,10 +68,10 @@ def configure(self, parser):
                                         help='Specify an identity file with '
                                         'a private key instead of a password '
                                         '(Example: .pem files from Amazon EC2)')
-        self.remote_parser.add_argument('--remote-no-copy',
-                                        dest='remote_no_copy',
+        self.remote_parser.add_argument('--remote-copy',
+                                        dest='remote_copy',
                                         action='store_true',
-                                        help="Don't copy tests and use the "
+                                        help="Copy tests and use the "
                                         "exact uri on guest machine.")
```

Comment:
Thinking about it, seems like it makes sense for the previous message, because we would look for the test in the exact path provided as url. But now with the inverted logics, it makes no sense to say that anymore.